### PR TITLE
ftx: handle errors of toLocalSymbol

### PR DESCRIPTION
**Pull request by jnlin**

### CODE REVIEW

File: pkg/exchange/ftx/convert.go [hunk 1]
```diff
@@ -21,6 +22,9 @@ func toGlobalSymbol(original string) string {
 }
 
 func toLocalSymbol(original string) string {
+	if "" == symbolMap[original] {
```
Comment (c9s) [inline pkg/exchange/ftx/convert.go:25]: I wonder if we can use go:generate to generate a static lookup map here

File: pkg/exchange/ftx/convert.go [hunk 2]
```diff
@@ -21,6 +22,9 @@ func toGlobalSymbol(original string) string {
...
+		panic(errors.New("Fatal Error: Can not find FTX's symbol mapping table"))
```
Comment (c9s) [inline pkg/exchange/ftx/convert.go:26]: errors use lower cases

Comment (jnlin) [reply pkg/exchange/ftx/convert.go]: fixed

File: pkg/exchange/ftx/convert.go (hunk 1 above)
Comment (jnlin) [reply pkg/exchange/ftx/convert.go]: We should handle symbols without local symbols (ex. BTC-PERP)

**Comment by jnlin** [comment]
Will send another PR

**Pull request closed without merging**